packages/design-system/package.json

```diff
@@ -63,7 +63,7 @@
     "node-notifier": "^10.0.1",
     "sass": "^1.58.0",
     "sass-loader": "^13.2.0",
-    "storybook": "^7.0.0-beta.46",
+    "storybook": "^7.0.8",
     "storybook-addon-designs": "^6.3.1",
```

Comment:
can we also please update this to `7.0.0-beta.2` to have a version compatible with storybook 7?

Reply:
I don't think we are using this addon anywhere. at least I could not find any configuration for it, besides it simply being added to `main.js`.
Removing this might help get rid of some of the storybook 6 related dependencies.

Reply:
Me neither, removed